source/help/messaging/mentioning-teammates.rst

```diff
@@ -50,6 +50,24 @@ You can ignore channel-wide mentions in specific channels in the **Channel Menu
 .. code-block:: none
 
   @here can someone do a quick review of this?
+  
+@groupname (Beta)(E20)
+~~~~~~~~~~
+
+System Admins can enable mentions for `LDAP synced groups <https://docs.mattermost.com/deployment/ldap-group-sync.html>`_ via the Group Configuration page.
+
+**Note**: This functionality is not yet supported on mobile.
+
+Once enabled for a specific group, users can mention and notify the entire group in a channel (similar to @channel or @all). Members of the group in that channel will receive a notification. If members of the group mentioned are not members of the channel, the user who posted the mention is prompted to invite them.
+
+Group mention identifiers (slugs) use the LDAP group name by default. You can customize/rename the slug via **System Console > Groups (Beta)**. Select **Edit** next to the group you want to edit.
```

Comment:
```suggestion
Group mention identifiers (slugs) use the LDAP group name by default. You can customize/rename the slug via **System Console > Groups (Beta)**. Select **Edit** next to the group you want to edit.
```
@thefactremains, is this correct? I wasn't able to confirm on my side and am wondering whether it was an issue on my side or the steps I provided are inaccurate. :)

Reply:
@thefactremains, should we leave this in here or remove it?